.circleci/config.yml

```diff
@@ -472,6 +490,9 @@ workflows:
                         branches:
                             only:
                                 - main
+            - bits-percy-visual-test:
+                  requires:
+                      - bits-build
```

Comment:
Should we add the same branch filter as the one used by `bits-visual-test` starting at line 489?

Reply:
makes sense, let me change that